tsconfig.json

```diff
@@ -9,15 +9,10 @@
     "outDir": "build",
     "jsx": "react-jsx",
     "strict": true,
-    "target": "es2021",
-    "types": [], //reconsider this later
+    "target": "es2023",
     "skipLibCheck": true, //reconsider this later
     "allowSyntheticDefaultImports": true,
     "baseUrl": "./frontend/javascripts",
-    "lib": [
```

Comment:
If `lib` is not set, TS uses a set of default libs + whatever `target` was specified.
https://www.typescriptlang.org/tsconfig/#lib